embedded_gems/pathogen/app/components/pathogen/datepicker/calendar/_footer.html.erb

```diff
@@ -0,0 +1,28 @@
+<div>
+  <% unless min_date.nil? %>
+    <div
+      class="text-sm text-center text-slate-900 dark:text-white"
+      data-pathogen--datepicker--calendar-target="minDateMessage"
+    >
+    </div>
+  <% end %>
+  <div class="flex space-x-2 rtl:space-x-reverse mt-2">
+    <button
+      type="button"
+      class="button button-primary w-1/2"
```

Comment:
Can you make this a `button-default` nothing about it needs to be the primary button